Automattic/Gravatar-SDK-Android
# QuickEditor: Migrate to Implicit OAuth flow

Closes #288

### Description

This PR changes the `response` to `token` from `code`. This enabled the implicit OAuth flow where we receive the token directly in the Fragment part of the redirect URL.

I've updated the secrets with the new whitelisted redirect URL so that the generated demo app works fine.

### Testing Steps

Test the OAuth flow in the QuickEditor - make sure you use `wpcom-local-dev` scheme.
Test the same for the demo app built for this PR.
